suites/reef/rados/tier-2_rados_test-brownfield.yaml

```diff
@@ -23,7 +23,7 @@ tests:
                 verbose: true
               args:
                 rhcs-version: 6.1
-                release: rc
+                release: z6
```

Comment:
Add a comment as to why we are installing z6. 

Add comment saying that issue gets reproduced only in this release.